doc/GUIDE.md

```diff
@@ -0,0 +1,822 @@
+# Jextract Guide
+
+The jextract tool parses header (.h) files of native libraries, and generates Java code,
+called 'bindings', which  use the Foreign Function and Memory API (FFM API) under the hood,
```

Comment:
maybe add a link for FFM to the final JEP, so that readers who want to know more can do so?